src/core/transactions/TransactionView.ts

```diff
@@ -137,25 +136,39 @@ export abstract class TransactionView<T extends Transaction> {
         key: 'signer_public_key',
         value: (this.transaction.signer && this.transaction.signer.publicKey) || undefined,
       },
-    ].filter((pair) => pair.value)
+    ]
+      .concat(this.getFeeDetailItem())
+      .filter((pair) => pair.value)
   }
 
-  protected getFeeDetailItem(): TransactionDetailItem {
+  protected getFeeDetailItem(): TransactionDetailItem[] {
     if (this.transaction.isConfirmed()) {
-      return {
-        key: 'paid_fee',
-        value: this.transaction,
-        isPaidFee: true,
-      }
+      return [
+        {
+          key: 'paid_fee',
+          value: this.transaction,
+          isPaidFee: true,
+        },
+      ]
     } else {
-      return {
-        key: 'max_fee',
-        value: {
-          amount: this.transaction.maxFee.compact() || 0,
-          color: 'red',
+      return [
+        {
+          key: 'max_fee',
+          value: {
+            amount: this.transaction.maxFee.compact() || 0,
+            color: 'red',
+          },
+          isMosaic: true,
+        },
+        {
```

Comment:
does this **have** to be added for **all** transaction types ? i.e. it is being added to the generic `TransactionView`, while rental fees are available only for a small count of transaction types, would there be a way to extend this method by child classes that would indeed have rental fees attached, only?